tools/wake/main.cpp

```diff
@@ -597,12 +597,12 @@ int main(int argc, char **argv) {
 
   std::vector<std::string> wakefilenames;
   {
-      struct timeval start, stop;
-      gettimeofday(&start, 0);
-      wakefilenames = find_all_wakefiles(enumok, clo.workspace, clo.verbose, libdir, ".", user_warn);
-      gettimeofday(&stop, 0);
-      double delay = (stop.tv_sec - start.tv_sec) + (stop.tv_usec - start.tv_usec) / 1000000.0;
-      wcl::log::info("Find wakefile took %f seconds", delay).urgent()();
+    struct timeval start, stop;
+    gettimeofday(&start, 0);
+    wakefilenames = find_all_wakefiles(enumok, clo.workspace, clo.verbose, libdir, ".", user_warn);
+    gettimeofday(&stop, 0);
+    double delay = (stop.tv_sec - start.tv_sec) + (stop.tv_usec - start.tv_usec) / 1000000.0;
+    wcl::log::info("Find wakefile took %f seconds", delay).urgent()();
```

Comment:
Why did the formatting here change?

Reply:
I'm pretty sure I manually "formatted" it and that manual format was just incorrect